Formula/cvs-fast-export.rb

```diff
@@ -5,6 +5,10 @@ class CvsFastExport < Formula
   sha256 "af58e16667c6e02e8431ff666effe306d3b30086fab483170257890da1afc21b"
   license "GPL-2.0-or-later"
 
+  livecheck do
+    url "https://gitlab.com/esr/cvs-fast-export.git"
```

Comment:
```suggestion
    url :homepage
    regex(/href=.*?cvs-fast-export[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Reply:
See my comment below - I was able to get `HEAD` working

Reply:
I've updated the livecheck to use `:homepage`